From pavel@ucw.cz  Mon Aug  7 16:59:03 2006
Date: Tue, 8 Aug 2006 01:37:15 +0200
From: Pavel Machek <pavel@suse.cz>
To: Greg KH <gregkh@suse.de>, Andrew Morton <akpm@osdl.org>
Cc: Shem Multinymous <multinymous@gmail.com>,
	Robert Love <rlove@rlove.org>, Jean Delvare <khali@linux-fr.org>,
	Alan Cox <alan@lxorguk.ukuu.org.uk>, linux-kernel@vger.kernel.org,
	hdaps-devel@lists.sourceforge.net
Subject: pr_debug() should not be used in drivers
Message-ID: <20060807233715.GL2759@elf.ucw.cz>
Content-Disposition: inline


pr_debug() should not be used from drivers, add comment saying that.

Signed-off-by: Pavel Machek <pavel@suse.cz>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 include/linux/kernel.h |    1 +
 1 file changed, 1 insertion(+)

--- gregkh-2.6.orig/include/linux/kernel.h
+++ gregkh-2.6/include/linux/kernel.h
@@ -210,6 +210,7 @@ extern enum system_states {
 extern void dump_stack(void);
 
 #ifdef DEBUG
+/* If you are writing a driver, please use dev_dbg instead */
 #define pr_debug(fmt,arg...) \
 	printk(KERN_DEBUG fmt,##arg)
 #else
